### ISSUE TITLE
QuotaExceededError: The quota has been exceeded.

### ISSUE THREAD

**Issue by darkroastcreative** [lufinkey/lufinkey.github.io#1]
When attempting to browse the site, the load process fails while trying to load `/system/plib/sass.js`, citing a `QuotaExceededError`.

```
failed to download /system/plib/sass.js
QuotaExceededError: The quota has been exceeded.
```
<img width="1326" alt="screen shot 2018-10-31 at 9 18 53 pm" src="https://user-images.githubusercontent.com/5994218/47827170-94e6a380-dd52-11e8-892e-bb204f2249da.png">

**Comment by lufinkey** [comment]
Yeah this seems to happen on certain browsers. I have a rework in a different branch I'm gonna fix it on eventually

**Comment by lufinkey** [comment]
It's basically because I'm using local storage which has a low limit on certain browsers. I need to store more of the dependency libraries in memory instead of in local storage
